ansible/roles/stack-sunbird/templates/dhiti-service.env

```diff
@@ -0,0 +1,85 @@
+APPLICATION_PORT_NUMBER={{dhiti_application_port_number}}
+APPLICATION_HOST_NAME={{dhiti_application_host_name}}
+APPLICATION_BASE_URL={{dhiti_application_base_url}}
+NODE_ENV={{dhiti_node_env}}
+APIDOC_URL={{dhiti_apidoc_url}}
+APIDOC_PATH={{dhiti_apidoc_path}}
+
+#Cloud storage
+CLOUD_STORAGE={{dhiti_cloud_storage}}
+BUCKET_NAME={{dhiti_bucket_name}}
+
+#Druid configuration
+DRUID_HOST={{dhiti_druid_host}}
+DRUID_PORT={{dhiti_druid_port}}
+OBSERVATION_DATASOURCE_NAME={{dhiti_observation_datasource_name}}
+ASSESSMENT_DATASOURCE_NAME={{dhiti_assessment_datasource_name}}
+ENROLLMENT_DATASOURCE_NAME={{dhiti_enrollment_datasource_name}}
+TELEMETRY_DATASOURCE_NAME={{dhiti_telemetry_datasource_name}}
+EVIDENCE_DATASOURCE_NAME={{dhiti_evidence_datasource_name}}
+SURVEY_DATASOURCE_NAME={{dhiti_survey_datasource_name}}
+SURVEY_EVIDENCE_DATASOURCE_NAME={{dhiti_survey_evidence_datasource_name}}
+OBSERVATION_EVIDENCE_DATASOURCE_NAME={{dhiti_observation_evidence_datasource_name}}
+CONTENT_REPORT_THRESHOLD={{dhiti_content_report_threshold}}
+ENTITY_SCORE_REPORT_THRESHOLD={{dhiti_entity_score_report_threshold}}
+OBSERVATION_SCORE_REPORT_THRESHOLD={{dhiti_observation_score_report_threshold}}
+ASSESSMENT_SUBMISSION_REPORT_THRESHOLD={{dhiti_assessment_submission_report_threshold}}
+EVIDENCE_THRESHOLD={{dhiti_evidence_threshold}}
+
+#Auth Config
+sunbird_keycloak_auth_server_url={{dhiti_sunbird_keycloak_auth_server_url}}
+sunbird_keycloak_realm={{dhiti_sunbird_keycloak_realm}}
+sunbird_keycloak_client_id={{dhiti_sunbird_keycloak_client_id}}
+sunbird_keycloak_public={{dhiti_sunbird_keycloak_public}}
+sunbird_cache_store={{dhiti_sunbird_cache_store}}
+sunbird_cache_ttl={{dhiti_sunbird_cache_ttl}}
+
+URL_PREFIX={{dhiti_url_prefix}}
+
+#ASSESSMENT service
+ASSESSMENT_SERVICE_APPLICATION_ENDPOINT={{dhiti_assessment_service_application_endpoint}}
+ASSESSMENT_SERVICE_BASE_URL={{dhiti_assessment_service_base_url}}
```

Comment:
These would be same everywhere. We should defined a default or harcode this to for this like `http://svc_name.{{namespace}}.svc.cluster.local`

Reply:
Done